integration/test_utils.go

```diff
@@ -91,6 +91,10 @@ func ConnectDaemons() error {
 	if err != nil {
 		return fmt.Errorf("failed to connect cri plugin: %v", err)
 	}
+	_, err = runtimeService.Status()
```

Comment:
I think we don't need this after we added `Load` during bootstrap. Or we should check readiness here.

Just check `Status` error doesn't make too much sense to me.

Reply:
yes will add that check

Reply:
It seems that you removed this. I'm also fine with that. :)